evalml/model_understanding/force_plots.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import shap
+
+from evalml.model_understanding.prediction_explanations import (
+    explain_predictions
+)
+
+
+def graph_force_plot(pipeline, rows_to_explain, training_data, y, matplotlib=False):
```

Comment:
+1, we should follow the convention our other graph methods follow unless there's a reason to do otherwise.

I think returning a plot is super useful for jupyter notebooks actually -- they'll auto-render the last returned value.

Reply:
Ah, I read a bit further. I see this is because both `graph_force_plot` and `force_plot` accept a list of rows to explain rather than a single row.

I suggest we define `force_plot` and `graph_force_plot` to accept a single row. Then define `force_plots` and `graph_force_plots` to accept multiple and to call the single-row methods.

Reply:
@dsherry I think I'm going to let #2287 handle the multi row case.  shap._force gives us back a different style of visualizer for multi-row explanations than it does for single row explanations.  How I handle that now is to provide the single row visualization for each row rather than the combined multi-row explanation.  Originally, before I used explain_predictions, `graph_force_plots` properly returned the visualization for both cases, making separate graphing functions unnecessary.  I hope after #2287 is complete, it will again! 

However, I will change the return to just be the plots rather than all the other junk.

Reply:
@freddyaboulton I am not entirely sure what this AdditiveForceVisualizer is.  It seems like it's a python wrapper for some javascript visualization.  Which makes me unsure if I can wrap it into a plotly figure.